### PR TITLE
Update runtime to 23.08

### PR DESCRIPTION
**Pull request by bbhtt**

### CODE REVIEW

File: com.pypyrev.linuxdcpp.json [hunk 1]
```diff
@@ -18,25 +18,31 @@
...
 			],
 			"cleanup": ["/include"]
 		},
 		{
 			"name": "libnotify",
 			"sources": [{
 					"type": "archive",
-					"url": "https://github.com/GNOME/libnotify/archive/refs/tags/0.7.9.tar.gz",
-					"sha256": "9bd4f5fa911d27567e7cc2d2d09d69356c16703c4e8d22c0b49a5c45651f3af0"
+					"url": "https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.2.tar.xz",
```
Comment (hfiguiere) [inline com.pypyrev.linuxdcpp.json]: isn't this API breaking?

Comment (bbhtt) [reply com.pypyrev.linuxdcpp.json]: Yea you're right Thanks

**Comment by bbhtt** [comment]
Runs

File: com.pypyrev.linuxdcpp.json (hunk 1 above)
Comment (bbhtt) [reply com.pypyrev.linuxdcpp.json]: Actually, looks like this wasn't API breaking

**Pull request merged**